docker/benchmarking-tools/Dockerfile

```diff
@@ -0,0 +1,143 @@
+
+# Install Picard
+
+# Please follow the below instructions to invoke picard when you are using this docker image:
+# java jvm-args -jar /usr/picard/picard.jar PicardToolName OPTION1=value1 OPTION2=value2...
+ENV picard_version 2.10.10
+WORKDIR /usr/picard
+ADD https://github.com/broadinstitute/picard/releases/download/${picard_version}/picard.jar ./picard.jar
+
+# Install Python3
+ENV PATH /usr/local/bin:$PATH
+RUN apt-get update \
+  && apt-get install -y python3-pip python3-dev \
+  && cd /usr/local/bin \
+  && ln -s /usr/bin/python3 python \
+  && pip3 install --upgrade pip
+
+# Install R and other dependencies
+RUN apt update && apt install -y \
+  	build-essential \
+	cmake automake \
+  	curl \
+  	gcc-multilib \
+  	git \
+  	libcurl4-openssl-dev \
+    libssl-dev \
+  	libboost-all-dev \
+  	libncurses5-dev \
+	libxml2-dev \
+  	libncurses5-dev \
+  	libboost-all-dev \
+	libbz2-dev \
+	liblzma-dev \
+	lsb-release \
+	samtools \
+	sudo \
+	wget \
+	zlib1g-dev \
+  gfortran
+
+RUN echo "deb http://http.debian.net/debian sid main" > /etc/apt/sources.list.d/debian-unstable.list \
+	&& echo 'APT::Default-Release "testing";' > /etc/apt/apt.conf.d/default
+ENV R_BASE_VERSION 3.4.4
+## Now install R and littler, and create a link for littler in /usr/local/bin
+## Also set a default CRAN repo, and make sure littler knows about it too
+RUN apt-get update \
+	&& apt-get install -t unstable -y --no-install-recommends \
+		littler \
+    r-cran-littler \
+		r-base=${R_BASE_VERSION}-* \
+		r-base-dev=${R_BASE_VERSION}-* \
+		r-recommended=${R_BASE_VERSION}-* \
+        && echo 'options(repos = c(CRAN = "https://cloud.r-project.org/"), download.file.method = "libcurl")' >> /etc/R/Rprofile.site \
+        && echo 'source("/etc/R/Rprofile.site")' >> /etc/littler.r \
+	&& ln -s /usr/share/doc/littler/examples/install.r /usr/local/bin/install.r \
+	&& ln -s /usr/share/doc/littler/examples/install2.r /usr/local/bin/install2.r \
+	&& ln -s /usr/share/doc/littler/examples/installGithub.r /usr/local/bin/installGithub.r \
+	&& ln -s /usr/share/doc/littler/examples/testInstalled.r /usr/local/bin/testInstalled.r \
+	&& install.r docopt \
+	&& rm -rf /tmp/downloaded_packages/ /tmp/*.rds \
+	&& rm -rf /var/lib/apt/lists/*
+
+# Install python packages
+RUN pip3 install \
+    crimson==0.3.0 \
+    HTSeq==0.9.0 \
+    matplotlib==2.1.0 \
+    numpy==1.12.0 \
+    pandas==0.20.3 \
+    pysam==0.12.0.1 \
+    requests==2.18.4 \
+    scipy==0.18.1 \
+    sctools==0.1.4 \
+    tables==3.4.2 \
+    google-cloud-storage \
+    git+git://github.com/HumanCellAtlas/pipeline-tools.git
+
+# Fix cannot import name 'opentype' error
+RUN pip3 install --upgrade google-auth-oauthlib
+# Mount Google Cloud secret key
+ADD broad-dsde-mint-dev-analysis-tools.json /usr/secrets/broad-dsde-mint-dev-analysis-tools.json
+
+# Set the environment variable for Python
+ENV GOOGLE_APPLICATION_CREDENTIALS=/usr/secrets/broad-dsde-mint-dev-analysis-tools.json
```

Comment:
Are you downloading files from a bucket inside R? It would be preferable to pass those file paths to the task and let Cromwell / Pipelines API localize them for you. That way you wouldn't need to bake credentials into this docker image.